stdppx/stdppx.ml

```diff
@@ -1,5 +1,5 @@
-module Caml = Stdlib
-open Caml
+module Caml = Stdlib [@@deprecated "[since 2023-06] use Stdlib instead"]
```

Comment:
By default, the deprecation alert is an error, isn't it? If so, I'd wait for the creation of the `5.1.0` bump PPX universe (i.e. not the upcoming release, but the one after). What do you think?

Reply:
sounds fine

Reply:
Are we ready for me to release this now?

Reply:
With "to release this" you mean "to merge this", right?

We've decided not to bump the AST to `5.1.0` and instead to wait for `5.2.0` since `5.1.0` hasn't introduced any new syntax feature that's reflected in the AST. So, if we wanted to wait for the next creation of a PPX universe, we'd have to wait for the `5.2.0` bump.

I'd say, instead let's just merge this now and be careful in the next opam release.

Just to make sure I'm not missing anything: The `Caml` module was just needed for something Jane Street internal, right? I think it got introduced when removing the dependencies on `base` and `stdio`, so it's probably related to that. Do you remember?